README.md

```diff
@@ -2,3 +2,10 @@ totem-application
 =================
 
 Application that allows to create totems to be included in other pages.
+
+## Building and installing
```

Comment:
`Building and installing from source` because don't forget that you may release it in the future and then, the installation will be done through the Extension Manager.